bionlp/utils/monitoring.py

```diff
@@ -47,6 +47,9 @@ def get_init_data():
 
 
 def add_iteration_data(data_type, data_dict):
+    global data
```

Comment:
Maybe I'm wrong, but since the loop is outside of python now it should be enough to simply declare `data = get_init_data()` outside of this function call, no? I don't like the `global data` here...